zkevm-circuits/src/evm_circuit/util/common_gadget.rs

```diff
@@ -445,3 +452,212 @@ impl<F: Field> TransferGadget<F> {
         Ok(())
     }
 }
+
+#[derive(Clone, Debug)]
+pub(crate) struct CommonCallGadget<F, const IS_ERROR: bool> {
```

Comment:
suggestion:  rename `IS_ERROR` to `IS_OOG_ERROR` we have merged other kind of error(insufficient balance error) into CallOp, in the future will probably merge depth overflow error into it. so make the error more clear !

Reply:
fixed in [7596a17](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1023/commits/7596a17e82b7f5d61b7611aca985419b4c2bb5a9)